pycatchmod/__main__.py

```diff
@@ -85,11 +86,19 @@ def run(ctx, parameters, rainfall, pet, output, output_key, rainfall_key, pet_ke
 
     # run catchmod
     print("Running catchmod...")
-    flows = run_catchmod(catchment, rainfall.values, pet.values, dates=dates)
+    flows = run_catchmod(catchment, rainfall.values, pet.values, dates=dates, output_total=total)
     print("Shape:", flows.shape)
 
-    df = pandas.DataFrame(flows, index=dates)
-    df.index.name = "Date"
+    if total:
+        df = pandas.DataFrame(flows, index=dates)
+        df.index.name = "Date"
+    else:
+        flows = flows.reshape((len(dates), len(catchment.subcatchments) * num_scenarios), order='F')
```

Comment:
Is this ordering=F correct here? It doesn't mess up the dates?

Reply:
yep - can't say i really understand the difference between C-like and Fortran-like indexing but seems to work :-)

Reply:
I would expect we should always be using the C ordering, unless we're payin passing arrays to Fortran, which we are not.

Reply:
Everything should be C ordering here. I think the correct solution is to swap the order of the levels of the multi-index. That should give the right output. 

Alternatively swap the order of the shape in the `run_catchmod` function.

```
flows = np.zeros([M2, N, len(C.subcatchments)])
...
for j in range(M2):
    ...
    for scenario in range(N):
        flows[j, scenario, :] = outflow[:, scenario]
```

This could do away with the loop actually ...

```
flows = np.zeros([M2, N, len(C.subcatchments)])
...
for j in range(M2):
    ...
    flows[j, ...] = outflow.T
```